src/Composer/PackageManager.php

```diff
@@ -499,7 +499,7 @@ private function linkConfig($name)
 
         // Check if we have a config file, and if it's readable. (yet)
         $configfilepath = $this->app['resources']->getPath('extensionsconfig/' . $configfilename);
-        if (is_readable($configfilepath)) {
+        if (is_readable($configfilepath) && !$this->app->isCli()) {
```

Comment:
```shell
[Symfony\Component\Routing\Exception\RouteNotFoundException]
  Unable to generate a URL for the named route "fileedit" as such route does not exist.
```
I don't know why, but the `fileedit` route was not defined. Any way, in command line this data not make sense to show :wink: